compactor/revision.go

```diff
@@ -17,6 +17,7 @@ package compactor
 import (
```

Comment:
no need to refactor revision compactor code? it is work as expected? we can refactor this in the future pr.
let's make this pr at a minimum?